### DISCUSSION TITLE
Email verification code activation

### DISCUSSION THREAD

**Discussion by wolone** [masterking32/WoWSimpleRegistration#96] [Ideas]
Do you want to consider adding the activation method of sending verification code by registered e-mail

**Comment by xclid** [comment]
Sounds good

**Comment by ex1dreamer** [comment]
please implement. without it users tend to abuse registration system and create multiple accounts with fake e-mails

**Comment by VoidLock92** [comment]
I would agree 100%

**Comment by TheInfamousNA** [comment]
This is most definitely something I would like to see as well
